CsvParser/csvParser.go

```diff
@@ -87,12 +76,18 @@ func CsvParser(filePath string, numIncidents ...int) []incident.DefaultIncident
 
 		ASN := strings.Split(rec[1], " ")[1]
 
+		testsAffected, err := strconv.Atoi(rec[0])
+
 		// Make an instance of a DefaultIncident that is compatible with the Incident interface
 		defaultIncident := new(incident.DefaultIncident)
 		defaultIncident.MakeIncidentData(goodStartTime, goodEndTime, timeStart, timeEnd, avgGoodDS, avgBadDS, ASN, locationString, severity, testsAffected)
 
-		defaultIncidents = append(defaultIncidents, *defaultIncident)
-
+		// Not all the rows in the csv are for incident data. The title row makes a good example for that
+		// We don't want to treat them as incidents
```

Comment:
The style of this comment does not match the rest of the file. Reword to say something like "Do not treat the title row (or any other row where testsAffected is not an integer) as an incident".